daisy_workflows/image_build/enterprise_linux/rhel_7_4_sap.wf.json

```diff
@@ -33,6 +33,7 @@
         "Path": "./enterprise_linux.wf.json",
         "Vars": {
           "el_release": "${rhel_release}",
+          "kickstart_config": "./kickstart/rhel_7_4_sap.cfg",
```

Comment:
7.4 is EOL and we can remove it entirely. 7.6 is currently the lowest version.

Reply:
i was trying to keep in line with the 'one change per PR' policy that's been suggested :) follow up changes include removing unused args and pruning now-unused workflows